apps/discovery_api/test/integration/docker-compose.yaml

```diff
@@ -113,6 +113,7 @@ services:
     - "9300:9300"
     environment:
     - "discovery.type=single-node"
+    - "bootstrap.system_call_filter=false"
```

Comment:
What does this do?

Reply:
While testing, I found that the elasticsearch would just become unhealthy after 5-6 minutes. This is to help those with M1 chips to not let that happen. Was something people have offered as a solution to the specific error presented.